include/d/actor/d_a_obj_doguu_demo.h

```diff
@@ -10,12 +12,19 @@ class daObjDoguuD_c : public fopAc_ac_c {
     inline BOOL _draw();
     inline BOOL _execute();
 
-    void CreateHeap();
+    BOOL CreateHeap();
     void CreateInit();
     void set_mtx();
 
 public:
-    /* Place member variables here */
+    /* 0x290 */ request_of_phase_process_class mPhs;
+    /* 0x298 */ J3DModel* mpModel;
+    /* 0x29C */ dBgW* mpBgW;
+    /* 0x2A0 */ Mtx mtx;
+    /* 0x2D0 */ u8 mFlag;  // this field seems to be set to 1 only when initalized with CreateInit,
+                           // can probably be a BOOL
```

Comment:
it can't be `BOOL` (which is a 32-bit type) but it might be `bool` (which is an 8-bit type)
(yes the devs used both types of booleans, and getting the type right affects matches a lot)